packages/frontend/src/stores/messagelist.ts

```diff
@@ -160,6 +185,11 @@ export function useMessageList(accountId: number, chatId: number) {
   return { store, state, fetchMoreTop, fetchMoreBottom }
 }
 
+/*
+ * Simply returns a subarray of items, from start to end
+ *
+ * @return {T[]} The subarray from start to end
+ */
 function getView<T>(items: T[], start: number, end: number) {
```

Comment:
```suggestion
 * Simply returns a subarray of items, from start to end
 */
function getView<T>(items: T[], start: number, end: number): T[] {
```

Reply:
Hmm I didn't know I can omit types in docstrings. Will commit it